manifests/params.pp

```diff
@@ -43,22 +43,11 @@
 
   case $facts['os']['name'] {
     /^(Debian|Ubuntu)$/: {
-      case $facts['os']['release']['full'] {
-        /^(8.*|14\.04)$/: {
-          $package_name          = 'squid3'
-          $service_name          = 'squid3'
-          $config                = '/etc/squid3/squid.conf'
-          $access_log            = 'daemon:/var/log/squid3/access.log squid'
-          $squid_bin_path        = '/usr/sbin/squid3'
-        }
-        default: {
-          $package_name          = 'squid'
-          $service_name          = 'squid'
-          $config                = '/etc/squid/squid.conf'
-          $access_log            = 'daemon:/var/log/squid/access.log squid'
-          $squid_bin_path        = '/usr/sbin/squid'
-        }
-      }
+      $package_name          = 'squid'
```

Comment:
Do we want to pull out the ones that are always the same for all 3 branches of the `case`?

Reply:
done